<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1303</width>
    <height>778</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>120</y>
      <width>741</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutMain">
     <item>
      <layout class="QGridLayout" name="gridLayoutMain"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1303</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_OpenDatabase">
   <property name="text">
    <string>&amp;Open Database</string>
   </property>
  </action>
  <action name="action_NewDatabase">
   <property name="text">
    <string>&amp;New Database...</string>
   </property>
  </action>
  <action name="action_DisplayDatabaseDrivers">
   <property name="text">
    <string>Display Database Drivers</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
